project/project-info.conf

```
@@ -112,4 +112,14 @@ project-info {
       }
     ]
   }
+  loglevels-log4j2: ${project-info.shared-info} {
+    title: "Dynamic Log Levels"
```

I think this title should be unique, so suffix it with specific logger (and please add "Logback" to the one above as well)
```suggestion
    title: "Dynamic Log Levels Log4j 2"
```